mmv1/products/compute/terraform.yaml

```diff
@@ -1961,6 +1961,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         is_set: true
         custom_flatten: templates/terraform/custom_flatten/preserved_state_disks.go.erb
         custom_expand: templates/terraform/custom_expand/preserved_state_disks.go.erb
+      preservedState.internalIp: !ruby/object:Overrides::Terraform::PropertyOverride
+        is_set: true
```

Comment:
`is_set` is probably not needed here as it's already defined as map

Reply:
Sure I will take it out.